lib/team.rb

```diff
@@ -0,0 +1,31 @@
+class Team
+  attr_reader :country, :players, :eliminated, :player_position,
+              :add_player, :players_by_position
+
+  def initialize(country)
+    @country = country
+    @players = []
+    @player_position = []
+    @eliminated = false
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def eliminated
```

Comment:
I really like this method, but think about naming it something a bit more distinct so that it doesn't get confused with the method `eliminated?`.